covbot.py

```diff
@@ -597,6 +599,24 @@ def _short_location(self, location: str, length=int(12)) -> str:
         c.format = "org.matrix.custom.html"
         await e.respond(c, markdown=True, allow_html=True)
 
+    @command.new('risks', help=HELP['risks'][1])
+    @command.argument("age", pass_raw=True, required=True)
+    async def risks_handler(self, event: MessageEvent, age: str) -> None:
+        # source : https://www.desmos.com/calculator/v0zif7tflm
+        age = int(age)
+        if age < 0 or age > 110:
+            await self._respond(event, "Age must be between 0 and 110")
+            return
+        death_rate = max(0, -0.00186807 + 0.00000351867 * age ** 2 + (2.7595 * 10 ** -15) * age ** 7)
+        intensive_care_rate = max(0, -0.0572602 - -0.0027617 * age)
+        hospitalization_rate = max(0, -0.0730827 - age * -0.00628289)
+        survival_rate = 1 - death_rate
+        response = '\n'.join([
```

Comment:
I would omit the manual newline, and have changed the wording very slightly:


```suggestion
        response = (
            "Based on reports, infection with SARS-CoV-2 in a {} year old has an average {:.1%} survial chance;"
            " with an average risk of hospitalization of {:.1%}, risk of needing intensive care of {:.1%},"
            " and {:.1%} chance of death."
        ).format(age, survival_rate, hospitalization_rate, intensive_care_rate, death_rate)
```

Which looks like:

![covrisks2](https://user-images.githubusercontent.com/7524620/77597599-4d3c5b00-6ef7-11ea-94b1-c4ec738ce272.jpg)

Reply:
* risk of death lol

Reply:
noice. its very clean code. thanks !